scripts/release.sh

```diff
@@ -119,6 +118,23 @@ main() {
     exit 1
   fi
 
+  # Check that gh is installed and logged in.
+  log_callout "Check gh installation"
+  if ! command -v gh >/dev/null; then
+    log_error "Cannot find gh. Please follow the installation instructions at https://github.com/cli/cli#installation"
+    exit 1
+  fi
+  local gh_version
+  gh_version=$(gh --version | head -n1 | awk 'match($0,/([0-9]+\.){2}[0-9]+/) {print substr($0,RSTART,RLENGTH)}')
+  if [ "$(cut -d. -f1 <(echo "${gh_version}"))" -lt "2" ] || [ "$(cut -d. -f2 <(echo "${gh_version}"))" -lt "57" ]; then
+    log_error "The minimum version required for gh is 2.57.0, got ${gh_version}"
```

Comment:
Curious - Why do we have a dependency on this version? Is it purely from a standpoint of what we have tested?

Reply:
I was having issues with `gh auth status`'s exit code (the exit code was 0 even when it failed). I swear I saw a pull request/a release note for this behavior in 2.57.0 (which is also the latest version). But I just tested again with multiple versions and went back all the way to 2.37.0, and it works as expected.

So, I'm fine removing this check.